syncopy/nwanalysis/connectivity_analysis.py

```diff
@@ -101,17 +101,15 @@ def connectivityanalysis(data, method="coh", keeptrials=False, output="abs",
         Frequency-window ``[fmin, fmax]`` (in Hz) of interest. The
         `foi` array will be constructed in 1Hz steps from `fmin` to
         `fmax` (inclusive).
-    pad_to_length : int, None or 'nextpow2'
-        Padding of the (tapered) signal, if set to a number pads all trials
-        to this absolute length. E.g. `pad_to_length=2000` pads all
-        trials to 2000 samples, if and only if the longest trial is
-        at maximum 2000 samples.
-
-        Alternatively if all trials have the same initial lengths
-        setting `pad_to_length='nextpow2'` pads all trials to
-        the next power of two.
-        If `None` and trials have unequal lengths all trials are padded to match
-        the longest trial.
+    pad : 'maxperlen', float or 'nextpow2'
```

Comment:
This reads a little bumpy imo - I think the more "technical" phrasing in the docstring of process_padding makes the point a little better. I rephrased the paragraph based on it (PR commit incoming)

Reply:
Yep, wasn't happy with it either 👍